### DISCUSSION TITLE
Any example of positioning and resizing image?

### DISCUSSION THREAD

**Discussion by kollol-dev** [ueberdosis/tiptap#1208] [General Questions] [answered]
Is there any way or any working example that anyone know of that I can resize the uploaded image as well as can position the image to the left/center/right.

**Comment by hanspagel** [comment, marked as answer]
I tried to build a resizeable image extension but failed to find a good library to make images resizeable. It’s still on my list though.

Do you want to position the image left/center/right or do you want to let the text float around the image?

**Reply by kollol-dev** [reply]
Floating around the image would be the best option.

**Reply by laurensiusadi** [reply]
Someone made this with prosemirror https://glitch.com/edit/#!/toothsome-shoemaker?path=index.js%3A28%3A40
Maybe it will help you @hanspagel

**Reply by hanspagel** [reply]
Thanks! That’s a very barebones implementation and unfortunately doesn’t work on mobile. I hoped to find a library that does all that reliable. Still couldn’t find time to do research though.
